rmf_traffic/package.xml

```diff
@@ -11,8 +11,7 @@
   <build_depend>rmf_utils</build_depend>
   <build_export_depend>rmf_utils</build_export_depend>
 
-  <build_depend>libccd-dev</build_depend>
-  <build_depend>libfcl-dev</build_depend>
+  <build_depend>fcl</build_depend>
```

Comment:
I'm not sure what we can/should do about this to make it backwards compatible. One option might be to keep the old two `build_depend` tags in here for now. I don't think there's much harm in having redundant dependency declarations.

Reply:
ebe339e